src/main/java/io/codiga/plugins/jetbrains/utils/CodePositionUtils.java

```diff
@@ -30,26 +30,43 @@ public static String indentOtherLines(String code, int indentation) {
         for (int i = 1 ; i < codeArray.length ; i++){
             StringBuilder sb = new StringBuilder();
             for (int j = 0 ; j < indentation ; j++) {
-                sb.append(CHARACTER_SPACE);
+                sb.append(usesTabs ? CHARACTER_TAB : CHARACTER_SPACE);
             }
             sb.append(codeArray[i]);
             codeArrayUpdated.add(sb.toString());
         }
         return String.join(LINE_SEPARATOR, codeArrayUpdated);
     }
 
+    /**
+     * Know if the current line has tabs indentation
+     *
+     * @param lineOfCode - the line of code.
+     * @return - boolean response if it is
+     */
+    public static boolean detectIfTabs(String lineOfCode) {
+      return lineOfCode.indexOf(CHARACTER_TAB) != -1;
```

Comment:
That one will fail if you have a tab later in the line.

Example of code
```
"     val bla = "<tab here>foo"
```

We should instead replace with the following:

```
for (int i = 0 ; i < lineOfCode.length() ; i++) {
   if(lineOfCode.characterAt(i) == CHARACTER_TAB) {
      return true;
   }
   if(lineOfCode.characterAt(i) != CHARACTER_TAB) {
      return false;
   }
}
```

or just

```
return lineOfCode.length > 0 && lineOfCode.charAt(0) == CHARACTER_TAB;
```

Please also add some unit tests to make sure this logic is correct.